boards/arm/96b_ivy5661/doc/index.rst

```diff
@@ -0,0 +1,228 @@
+
+- UNISOC UWP5661 in 28nm package
+- ARM |reg| 32-bit Cortex |reg|-M4 Dual Core CPU
+- 416 MHz max CPU frequency
+- 4MB Flash
+- 832KB SRAM
+- IEEE802.11ac 2x2 Wi-Fi
+- Bluetooth 5
+- 4 User LEDs
+- Reset and User button
+- GPIO with external interrupt capability
+- UART (2)
+- I2C (1)
+- SPI (1)
+- I2S (1)
+
+Supported Features
+==================
+
+The Zephyr 96b_ivy5661 board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | system clock                        |
++-----------+------------+-------------------------------------+
+| ICTL      | on-chip    | second level interrupt controller   |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+
+The default board configuration can be found in the defconfig file:
+:zephyr_file:`boards/arm/96b_ivy5661/96b_ivy5661_defconfig`
+
+Connections and IOs
+===================
+
+LED
+---
+
+- LED5 / Power LED = VDD
+- LED6 / User LED  = GPIOA3
+- LED7 / User LED  = GPIOA1
+- LED7301 / User LED  = GPIOA2
+
+Push buttons
+------------
+
+- S4 / Reset Button = RST_N
+- S1 / User Button  = GPIOA0
+
+.. _Jumpers:
+
+Jumpers
+-------
+
+There are several jumpers on IVY5661, which are used for switching boot mode.
+
++--------+--------+--------+--------+----------------+-------------------------+
+| RFCTL8 | RFCTL7 | RFCTL6 | RFCTL5 |    Boot Mode   |        Description      |
++========+========+========+========+================+=========================+
+|   0    |   0    |   1    |   0    | ROM boot       | Update image from UART0 |
++--------+--------+--------+--------+----------------+-------------------------+
+|   0    |   0    |   0    |   1    | NOR Flash boot | XIP mode                |
++--------+--------+--------+--------+----------------+-------------------------+
+
+System Clock
+============
+
+96Boards IVY5661 can be driven by an internal oscillator as well as the main
+PLL clock. By default System clock is sourced by PLL clock at 416MHz, driven
+by internal oscillator.
+
+Serial Port
+===========
+
+On 96Boards IVY5661 Zephyr console output is assigned to UART0.
+Default settings are 115200 8N1.
+
+Programming and Debugging
+*************************
+
+Building
+========
+
+Here is an example for building the :ref:`hello_world` application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: 96b_ivy5661
+   :goals: build
+
+Flashing
+========
+
+96Boards IVY5661 can be flashed by two methods, one using the ROM
```

Comment:
delete this paragraph.  (It's repeated below, with a correction)

Reply:
Oops. Will remove it.